docs/content/preview/explore/_index.md

```diff
@@ -12,151 +12,183 @@ aliases:
   - /preview/explore/planet-scale/
   - /preview/explore/cloud-native/orchestration-readiness/
 type: indexpage
+showRightNav: true
 ---
 
-<div class="row">
-  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
-    <a class="section-link icon-offset" href="ysql-language-features/">
-      <div class="head">
-        <img class="icon" src="/images/section_icons/explore/fault_tolerance.png" aria-hidden="true" />
-        <div class="title">SQL features</div>
-      </div>
-      <div class="body">
-        SQL features in YSQL.
-      </div>
-    </a>
-  </div>
+The Explore section walks you through YugabyteDB's core features, with examples.
```

Comment:
Suggestion:
The Explore section introduces you to the YugabyteDB's features and provides examples.